app/experimenter/nimbus-ui/src/components/PageHome/Directory.tsx

```diff
@@ -0,0 +1,94 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useEffect, useState } from "react";
+import { useQuery } from "@apollo/client";
+import { GET_ALL_EXPERIMENTS } from "../../gql/experiments";
+import { GetAllExperiments_experiments } from "../../types/getAllExperiments";
+import { humanDate, addDaysToDate } from "../../lib/dateUtils";
+import PageLoading from "../PageLoading/index";
+import { getStatus } from "../../lib/experiment";
+
+// Placeholder directory until we get the real thing!
+const Directory: React.FunctionComponent = () => {
+  const { data, loading } = useQuery<{
+    experiments: GetAllExperiments_experiments[];
+  }>(GET_ALL_EXPERIMENTS);
+
+  if (loading) {
+    return <PageLoading />;
+  }
+
+  if (!data) {
+    return <div>No experiments found.</div>;
+  }
+
+  const { experiments } = data;
+
+  return (
+    <div>
+      <table className="table table-sm">
+        <thead>
+          <tr>
+            <th>Experiment</th>
+            <th>Type</th>
+            <th>Status</th>
+            <th>Start Date</th>
+            <th>End Date</th>
+            <th>Enrollment End Date</th>
+            <th>Results</th>
+          </tr>
+        </thead>
+        <tbody>
+          {experiments.map((experiment) => {
+            const {
+              name,
+              slug,
+              owner,
+              startDate,
+              proposedDuration,
+              proposedEnrollment,
+              endDate,
+              featureConfig,
+            } = experiment;
+            const status = getStatus(experiment);
+            return (
+              <tr key={slug}>
+                <td>
+                  <a href={`/experiments/nimbus/${slug}`}>{name}</a>
+                </td>
+                <td>
+                  {featureConfig ? featureConfig.slug : "Simple Experiment"}
+                </td>
+                <td>{getStatus(experiment)}</td>
+                <td>{owner.username}</td>
+                <td>{startDate ? humanDate(startDate) : "-"}</td>
+                <td>
+                  {endDate
+                    ? humanDate(endDate)
+                    : startDate
+                    ? humanDate(addDaysToDate(startDate, proposedDuration))
+                    : `+${proposedDuration} days`}
+                </td>
+                <td>
+                  {startDate
+                    ? humanDate(addDaysToDate(startDate, proposedEnrollment))
+                    : `+${proposedEnrollment} days`}
+                </td>
+                <td>
+                  {status.released && (
+                    <a href={`/experiments/nimbus/${slug}/results`}>Results</a>
```

Comment:
Same comment as above here re: `/experiments/` prefix (which should now 404 now that we've removed that path segment).